numba/parfor.py

```diff
@@ -0,0 +1,1246 @@
+from __future__ import print_function, division, absolute_import
```

Comment:
Can you add a highlevel description or literature reference for the transformations happening in this file?